### PR TITLE
[CELADON] Updating the Release Notes for WW40

### PR DESCRIPTION
**Pull request by swaroopbalan**
Signed-off-by: Swaroop Balan <swaroop.balan@intel.com>

### CODE REVIEW

File: source/release_notes.rst [hunk 1]
```diff
@@ -6,6 +6,71 @@ Release Notes
 .. contents:: :local:
 
 
+--------
+
+ww201840
+========
+
+================================  =====
+Build ID
```
Comment (ttzeng) [inline source/release_notes.rst]: The Build ID is missing?

File: source/release_notes.rst [hunk 2]
```diff
@@ -6,6 +6,71 @@ Release Notes
...
+* Analog Audio (MP3/AAC) playback on stereo wired Headset
+* Audio parameter framework for audio routing policy
+* Wi-Fi 802.11 a/b/g/n
+* Wired Ethernet support
+* Android Grub boot support
+* Android Kernel Flinger boot support
+* Thermal Daemon is enabled for CELADON
+* The HW media codec is enabled for CELADON.
+	HW codec capabilities supported : 
+		Decoder: h264, h265, vp8, vp9, jpeg
```
Comment (ttzeng) [inline source/release_notes.rst]: Line break required to start "Decoder ..." in a new line.

File: source/release_notes.rst [hunk 3]
```diff
@@ -6,6 +6,71 @@ Release Notes
...
+* HWC 2.0 support for Display
+* USB digital audio playback support
+* Analog Audio (MP3/AAC) playback on stereo wired Headset
+* Audio parameter framework for audio routing policy
+* Wi-Fi 802.11 a/b/g/n
+* Wired Ethernet support
+* Android Grub boot support
+* Android Kernel Flinger boot support
+* Thermal Daemon is enabled for CELADON
+* The HW media codec is enabled for CELADON.
```
Comment (ttzeng) [inline source/release_notes.rst]: This line is rendered in italics font, you may remove the ending dot for a try.

File: source/release_notes.rst [hunk 4]
```diff
@@ -6,6 +6,71 @@ Release Notes
...
+	HW codec capabilities supported : 
+		Decoder: h264, h265, vp8, vp9, jpeg 
+		Encoder: h264, h265(WIP, will ready in early WW32), jpeg
+
+• Media sdk 4K support is enabled on CELADON
```
Comment (ttzeng) [inline source/release_notes.rst]: Start the line with '*' to be consistent with the previous bullets. Same for the bullet below.

File: source/release_notes.rst [hunk 5]
```diff
@@ -6,6 +6,71 @@ Release Notes
...
+BT                             OK       File Transfer Protocol
+Audio over USB                 OK       MP3/AAC/MIDI/FLAC/WAV Audio playback
+Audio over 3.5mm jack          OK       MP3/AAC/MIDI/FLAC/WAV Audio playback
+Video                          OK       H263/ H264/ HEVC/VP9/MPEG4 video formats
+Adb connect over WIFI,Ethernet OK
+Display /Touch and Gesture     OK
+Storage/SD Card                OK
+Security                       OK
+Boot/Kernel                    OK
+Fastboot                       Not available
```
Comment (ttzeng) [inline source/release_notes.rst]: The table is not rendered completely, try to align the "Not available" to the 3rd column.

File: source/release_notes.rst [hunk 6]
```diff
@@ -6,211 +6,69 @@ Release Notes
...
-* H263/ H264/ HEVC/VP9/MPEG4 Video Playback using google video software codecs
 * Wi-Fi 802.11 a/b/g/n
 * Wired Ethernet support
-* Open CI infrastructure for automatic build to verify and upstream patches
 * Android Grub boot support
 * Android Kernel Flinger boot support
+* Thermal Daemon is enabled for CELADON
+* The HW media codec is enabled for CELADON
+* HW codec capabilities supported : 
+* Decoder: h264, h265, vp8, vp9, jpeg
```
Comment (slattann) [inline source/release_notes.rst]: Anybody tested these features?
If yes please let us know, our SQE team reported issues with these features. Thanks
-Sameer

Comment (swaroopbalan) [reply source/release_notes.rst]: Please check with Chen Tianmi

**Pull request merged**